multimedia/ffmpeg4/Makefile

```diff
@@ -0,0 +1,345 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ffmpeg4
+PKG_VERSION:=4.2.1
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=
+PKG_LICENSE:=GPL-2.0-or-later LGPL-2.1-or-later LGPL-3.0-only
+PKG_LICENSE_FILES:=COPYING.GPLv2 COPYING.GPLv3 COPYING.LGPLv2.1 COPYING.LGPLv3
+
+PKG_SOURCE:=ffmpeg-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.ffmpeg.org/releases/
+PKG_HASH:=cec7c87e9b60d174509e263ac4011b522385fd0775292e1670ecc1180c9bb6d4
+PKG_BUILD_DIR:=$(BUILD_DIR)/ffmpeg-$(PKG_VERSION)
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:= \
+  CONFIG_ffmpeg4_LICENSING \
+  CONFIG_ffmpeg4_ENC_FLV \
+  CONFIG_ffmpeg4_ENC_MJPEG \
+  CONFIG_ffmpeg4_ENC_MPEG4 \
+  CONFIG_ffmpeg4_ENC_PNG \
+  CONFIG_ffmpeg4_FEAT_DASH \
+  CONFIG_ffmpeg4_LIB_FDK-AAC \
+  CONFIG_ffmpeg4_LIB_LAME \
+  CONFIG_ffmpeg4_LIB_MBEDTLS \
+  CONFIG_ffmpeg4_LIB_OPENSSL \
+  CONFIG_ffmpeg4_LIB_LIBOPUS \
+  CONFIG_ffmpeg4_LIB_LIBVORBIS \
+  CONFIG_ffmpeg4_LIB_LIBX264 \
+  CONFIG_ffmpeg4_OPT_PERFORMANCE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ffmpeg4
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+  DEPENDS:=+libpthread +zlib @BUILD_PATENTED \
+    +ffmpeg4_FEAT_DASH:libxml2 \
+    +ffmpeg4_LIB_FDK-AAC:fdk-aac \
+    +ffmpeg4_LIB_LAME:lame-lib \
+    +ffmpeg4_LIB_MBEDTLS:libmbedtls \
+    +ffmpeg4_LIB_OPENSSL:libopenssl \
+    +ffmpeg4_LIB_LIBOPUS:libopus \
+    +ffmpeg4_LIB_LIBVORBIS:libvorbis \
+    +ffmpeg4_LIB_LIBX264:libx264
```

Comment:
This is causing a recursive dependency:
```
package/multimedia/ffmpeg4/Config.in:70:error: recursive dependency detected!
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
package/multimedia/ffmpeg4/Config.in:70:        symbol ffmpeg4_LIB_LIBX264 depends on PACKAGE_ffmpeg4
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:24262:   symbol PACKAGE_ffmpeg4 is selected by PACKAGE_ffmpeg4-ffprobe
For a resolution refer to Documentation/kbuild/kconfig-language.txt
subsection "Kconfig recursive dependency limitations"
tmp/.config-package.in:24304:   symbol PACKAGE_ffmpeg4-ffprobe depends on ffmpeg4_LIB_LIBX264
```

Reply:
Can't reproduce, works fine on Alpine Linux at least using master.

Reply:
I'm using master on Debian, and I get recursive dependencies...

Reply:
This seems to pop up on master now, I have no idea (or interest spending time) to work around limitations of the framework as it's just noise.

Reply:
To be more accurate about it, while it is harmless it most cases it may still cause issues but I have no idea how to fix it.